doc/developer/guide.md

```diff
@@ -341,35 +299,71 @@ acceptable for:
     entry. Changes to Materialize very rarely require changes in rust-dec, so
     maintaining the two separately does not introduce much overhead.
 
-## Code organization
+## Developer tools
 
-We break our code into crates primarily to promote organization of code by team,
-thereby introducing ownership and autonomy. As such, many crates are owned by a
-specific team (which does not preclude the existence of shared, cross-team
-crates).
+### Editors and IDEs
 
-Although the primary unit of code organization at the inter-team level is the
-crate, modules within a crate are also useful for code organization, especially
-because they are the level at which `pub` visibility operates.
+In principle, any text editor can be used to edit Rust code. There is nothing
+special about Materialize's codebase: any general Rust development setup should
+work fine.
```

Comment:
I wrote this paragraph, but I don't quite agree with it anymore. Materialize now _is_ more complicated than a typical Rust project, in that you can't just run `cargo run` or `cargo test` and expect things to work.

I think we can leave in the bit about any text editor being fine, but delete the (sadly no longer true) bit about how Mz is just a vanilla Rust binary.

```suggestion
In principle, any text editor can be used to edit Rust code, including Materialize.
```

Reply:
Ack, done!